android/app/build.gradle

```diff
@@ -270,6 +270,7 @@ dependencies {
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.1.0"
     implementation 'io.github.novacrypto:BIP39:2019.01.27'
     implementation 'com.google.android.play:core:1.8.2'
+    implementation 'com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava'
```

Comment:
Some random android bug: https://stackoverflow.com/questions/56639529/duplicate-class-com-google-common-util-concurrent-listenablefuture-found-in-modu